conf/modules.config

```diff
@@ -382,6 +382,18 @@ process {
         ]
     }
 
+    withName: HOST_REMOVAL {
+        ext.args= {[ "-m ${params.hostremoval_mode}",
+        "${meta_fastqs.single_end}" == false && "!${params.preprocessing_skippairmerging}" ? "-merged" : ""
```

Comment:
Jsut to confirm: mMerged also applies for single end data?

Reply:
@jfy133: The merged flag only applies for pair end reads that have been merged. So in here I am checking if meta_fastqs is paired_end and you haven't set the skip pair merging, meaning they were merged

Reply:
:+1: